Tests/SonarScanner.MSBuild.PostProcessor.Test/PostProcessorTests.cs

```diff
@@ -92,15 +92,15 @@ public void PostProc_NoProjectsToAnalyze_NoExecutionTriggered()
         context.TfsProcessor.AssertNotExecuted();
         context.Scanner.AssertNotExecuted();
         context.Logger.AssertErrorsLogged(0);
-        context.Logger.AssertSingleWarningExists("""Multi-file Analysis is enabled. If this was not intended, please set "/d:sonar.scanner.multiFileAnalysis=false" in the begin step.""");
+        context.Logger.AssertSingleWarningExists("""ScanAllAnalysis is enabled. If this was not intended, please set "/d:sonar.scanner.scanAll=false" in the begin step.""");
```

Comment:
I think the official term is "multi-language".
I don't think "ScanAll" is very readable in the logs, as the users will likely search for the term and in the documentation it will be "Multi-Language"

Reply:
(The parameter name is correct)